src/components/datagrid/column_sorting_draggable.tsx

```diff
@@ -70,13 +70,15 @@ export const EuiDataGridColumnSortingDraggable: FunctionComponent<EuiDataGridCol
           }`}
         >
           <EuiScreenReaderOnly>
-            <EuiI18n
-              token="euiColumnSortingDraggable.activeSortLabel"
-              default="{display} is sorting this data grid"
-              values={{ display }}
-            >
-              {(activeSortLabel: string) => <p>{activeSortLabel}</p>}
-            </EuiI18n>
+            <p>
+              <EuiI18n
+                token="euiColumnSortingDraggable.activeSortLabel"
+                default="{display} is sorting this data grid"
+                values={{ display }}
+              >
+                {(activeSortLabel: string) => <>{activeSortLabel}</>}
```

Comment:
optional nit, I believe you don't need a fragment at all - just a string is fine

```suggestion
                {(activeSortLabel: string) => activeSortLabel}
```